crud/common/utils.lua

```diff
@@ -375,6 +375,18 @@ function utils.get_bucket_id_fieldno(space, shard_index_name)
     return bucket_id_index.parts[1].fieldno
 end
 
+function utils.get_index_fieldno_map(index_parts)
```

Comment:
This 'index fieldno map' is not similar to the 'format fieldno map', defined just below. One maps from field names, another from field numbers. One maps to booleans, another to field numbers. Maybe we should use a different term there to reduce confusion. 'Fieldno mask', maybe.

Reply:
I would keep it as is.

Added description for `get_index_fieldno_map()`:
```
--- a/crud/common/utils.lua
+++ b/crud/common/utils.lua
@@ -375,6 +375,8 @@ function utils.get_bucket_id_fieldno(space, shard_index_name)
     return bucket_id_index.parts[1].fieldno
 end
 
+-- Build a map with field number as a keys and part number
+-- as a values using index parts as a source.
 function utils.get_index_fieldno_map(index_parts)
     dev_checks('table')
 
```